overrides/partials/version.html

```diff
@@ -4,7 +4,7 @@
 {% set repo_url = config.repo_url + "/releases/tag/" + git.tag %}
 {% endif %}
 
-<a href="{{ repo_url }}">
+<a href="/versions">
```

Comment:
This link is broken. That's related to the remark in the `TODO`. A relative link also won't work because that would be different from each page. I was thinking that maybe introducing a `base` element in the HTML would make sense, but then we'd have to adapt all other links in the site, too. I'm not sure what to do here.

Reply:
I agree that this is a problem. I am stuck with how to resolve it. Perhaps we should just remove it for now?

Reply:
That would probably be the best idea, reverting to the link to Github. We can then later see how to solve issues like these.

Reply:
I worked out something that I like better. Let's see what you think, @ifokkema.

I made https://hgvs-nomenclature/versions redirect to https://hgvs-nomenclature/stable/versions on RTD. This means that the absolute URL will always resolve to the most recent stable documentation version. IMO, that's 99% of the goal.

In branch deployments, the correct versions will be be accessible in the nav menu. The only thing that will be broken is that clicking the version tag in the upper right will take the user (a developer, likely) to the stable versions page. This seems like a small annoyance. 

And local development will always give the local version of the versions page.

The redirect is working now (but 404 because the versions page doesn't exist yet). If you agree, I'll leave the URL as /versions/.

Reply:
Sure, this will be OK! And now that the site is live, the link works.